windows/drivers/uwb/simulator/UwbSimulatorDdiCallbacksLrp.hxx

```diff
@@ -72,23 +72,23 @@ struct UwbSimulatorDdiCallbacksLrp
     DeviceSetConfigurationParameters(const std::vector<UwbDeviceConfigurationParameter> &deviceConfigurationParameters, std::vector<std::tuple<UwbDeviceConfigurationParameterType, UwbStatus>> &deviceConfigurationParameterResults) = 0;
 
     /**
-     * @brief Set the Application Configuration Parameters object
+     * @brief Get the Application Configuration Parameters object
      *
      * @param applicationConfigurationParameters
-     * @param applicationConfigurationParameterResults
      * @return UwbStatus
      */
     virtual UwbStatus
-    SetApplicationConfigurationParameters(const std::vector<std::unique_ptr<IUwbAppConfigurationParameter>> &applicationConfigurationParameters, std::vector<std::tuple<UwbApplicationConfigurationParameterType, UwbStatus, std::unique_ptr<IUwbAppConfigurationParameter>>> &applicationConfigurationParameterResults) = 0;
+    GetApplicationConfigurationParameters(std::vector<UwbApplicationConfigurationParameter> &applicationConfigurationParameters) = 0;
 
     /**
-     * @brief Get the Application Configuration Parameters object
+     * @brief Set the Application Configuration Parameters object
      *
      * @param applicationConfigurationParameters
+     * @param applicationConfigurationParameterResults
      * @return UwbStatus
      */
     virtual UwbStatus
-    GetApplicationConfigurationParameters(std::vector<UwbApplicationConfigurationParameter> &applicationConfigurationParameters) = 0;
+    SetApplicationConfigurationParameters(const std::vector<std::unique_ptr<IUwbAppConfigurationParameter>> &applicationConfigurationParameters, std::vector<std::tuple<UwbApplicationConfigurationParameterType, UwbStatus, std::unique_ptr<IUwbAppConfigurationParameter>>> &applicationConfigurationParameterResults) = 0;

```

Comment:
These didn't actually change, only their order was flipped and trhe difftool got confused.